Documentation/helm-ceph-cluster.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Note there is a section in advanced configuration for [using alternative namespaces](https://github.com/rook/rook/blob/master/Documentation/ceph-advanced-configuration.md#using-alternate-namespaces). How about adding a link in that section back to this helm chart as the solution to that problem?